src/setup.js

```diff
@@ -25,6 +25,8 @@ global.provDevice;
 global.options.deviceType = 'raspberrypi3'
 global.options.version = 'latest'
 
+global.auth_token = process.env.auth_token;
```

Comment:
Lets not create more global variables. The provisioning code that needs this token can access it on `process.env` directly anyways.